godot-macros/src/lib.rs

```diff
@@ -444,6 +446,32 @@ use crate::util::ident;
 /// #     fn init(base: godot::obj::Base<Self::Base>) -> Self { todo!() }
 /// # }
 /// ```
+///
+/// # Documentation
+///
+/// You can document your functions, classes, members, and signals with the `///` doc comment syntax.
```

Comment:
You could mention that the `register-docs` feature needs to be enabled for this to work, and that it only works for Godot 4.3+?

Reply:
![image](https://github.com/godot-rust/gdext/assets/70787919/29e501bd-6cb5-4584-a452-e429af6a2845)
hows this?

Reply:
Oh, good call, that's enough then 👍